cmd/link/link.go

```diff
@@ -3,20 +3,32 @@ package link
 import (
 	"context"
 	"fmt"
+	"time"
 
 	"github.com/rclone/rclone/cmd"
+	"github.com/rclone/rclone/fs/config/flags"
 	"github.com/rclone/rclone/fs/operations"
 	"github.com/spf13/cobra"
 )
 
+var (
+	expire = time.Minute * 15
```

Comment:
This should default to very large to be compatible with previous rclone versions - say 100 years!

Also using a type of `fs.Duration` here will give the user more options when configuring it including days, weeks absolute times etc! You'll need to cast it back to time.Duration for the call.

    expire = fs.Duration(time.Hour * 24 * 365 * 100)